### PR TITLE
Wicklow

### PR DESCRIPTION
**Pull request by deepakvk**
Thank you for your contribution to the API Builder documentation.

## Describe the changes

Enter a brief description of your changes to communicate to the reviewers what you changed and why.

## Checklist for contributors

Before submitting this PR, please make sure:

* [ ] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [ ] You have signed the [Axway CLA](https://cla.axway.com/)
* [ ] You have verified the technical accuracy of your change
* [ ] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [ ] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)
* [ ] You have verified that all status checks have passed

_Put an x in the boxes that apply. This is simply a reminder of what we are going to look for before merging your change._

### CODE REVIEW

File: content/en/docs/release_notes/wicklow.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+title: Wicklow release notes
+linkTitle: Wicklow
+description: 30 September 2022
+date: 2022-09-30
+Hide_readingtime: true
+---
+## Summary
+Fixed a bug with `@axway/requester` on calculating the accurate `Content-Length` of the request `body`.
```
Comment (alasdairhurst) [inline content/en/docs/release_notes/wicklow.md:9]: **issue:** comment made regarding this summary on the v5 docs and unresolved.

File: content/en/docs/release_notes/wicklow.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+---
+title: Wicklow release notes
+linkTitle: Wicklow
+description: 30 September 2022
```
Comment (alasdairhurst) [inline content/en/docs/release_notes/wicklow.md:4]: Release date needs updating here and on JIRA. The other releases that were targeted for this week need to be pushed back to 2 weeks time.

File: content/en/docs/release_notes/wicklow.md (hunk 1 above)
Comment (deepakvk) [reply content/en/docs/release_notes/wicklow.md:9]: @alasdairhurst , So is it ok to keep the Summary and Fixes doc same?

File: layouts/shortcodes/releasenotes/update.md [hunk 3]
```diff
@@ -0,0 +1,2 @@
+## Update
+Available updates can be found in the _Updates_ tab in the UI. For more information, including how to update the CLI, read the [Update guide](/docs/updates/update_guide/).
```
[the remark this answers is not in the document]
Comment (deepakvk) [reply layouts/shortcodes/releasenotes/update.md:2]: Fixed

File: content/en/docs/release_notes/wicklow.md (hunk 1 above)
Comment (deepakvk) [reply content/en/docs/release_notes/wicklow.md]: Fixed

File: content/en/docs/release_notes/wicklow.md (hunk 2 above)
Comment (deepakvk) [reply content/en/docs/release_notes/wicklow.md]: Done

**Pull request merged**